plugins/builtin/target/gce-mig/plugin/plugin.go

```diff
@@ -0,0 +1,201 @@
+package plugin
+
+import (
+	"context"
+	"fmt"
+	hclog "github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/nomad-autoscaler/plugins"
+	"github.com/hashicorp/nomad-autoscaler/plugins/base"
+	"github.com/hashicorp/nomad-autoscaler/plugins/target"
+	"github.com/hashicorp/nomad-autoscaler/sdk"
+	"github.com/hashicorp/nomad-autoscaler/sdk/helper/nomad"
+	"github.com/hashicorp/nomad-autoscaler/sdk/helper/scaleutils"
+	"google.golang.org/api/compute/v1"
```

Comment:
nit: could the imports be separated; stdlib top and other below, with the two sections separated by a newline and each section sorted alphabetically?